docs/articles_en/get_started/installing-openvino-overview/installing-openvino-linux-header/installing-openvino-yum.rst

```diff
@@ -38,6 +38,8 @@ Install OpenVINO™ Runtime on Linux From YUM Repository
          - OpenEuler 20.03 and 22.03
          - Anolis OS 8.6 and 8.8
          - CentOS Stream 8 and 9
+         - openSUSE 15.4 and openSUSE 15.5
```

Comment:
does openSUSE have yum / dnf package manager?

Reply:
> does openSUSE have yum / dnf package manager?

I made a mistake, corrected the list.